src/app/components/Billboard/BillboardCurationGrid/index.styles.ts

```diff
@@ -0,0 +1,134 @@
+import { css, Theme } from '@emotion/react';
+
+const styles = {
+  list: ({ isLite, mq }: Theme) =>
+    css({
+      listStyleType: 'none',
+      padding: 0,
+      margin: 0,
+      display: 'grid',
+      gap: '1rem',
+
+      [mq.GROUP_2_MAX_WIDTH]: {
+        gridTemplateColumns: '1fr',
+      },
+      [mq.GROUP_3_ONLY]: {
+        gridTemplateColumns: 'repeat(3, 1fr)',
+
+        '& > li:nth-of-type(4)': {
+          display: 'none',
+        },
+      },
+
+      [mq.GROUP_4_MIN_WIDTH]: {
+        gridTemplateColumns: 'repeat(4, 1fr)',
+      },
+
+      ...(isLite && {
+        li: {
+          '.promo-text': {
+            width: '100%',
+            paddingInlineStart: 0,
+          },
+        },
+      }),
+    }),
+
+  item: ({ spacings, mq, palette }: Theme) =>
+    css({
+      verticalAlign: 'top',
+      display: 'block',
+      width: '100%',
+
+      '.promo-image': {
+        [mq.GROUP_2_MAX_WIDTH]: {
+          width: '33%',
+          display: 'inline-block',
+          verticalAlign: 'top',
+        },
+        'div div:nth-child(2)': {
+          [mq.GROUP_1_MAX_WIDTH]: {
+            position: 'relative',
+          },
+          div: {
+            padding: `${spacings.FULL}rem`,
+            [mq.GROUP_4_MIN_WIDTH]: {
+              padding: '0.75rem',
+            },
+            svg: {
+              margin: 0,
+              [mq.GROUP_4_MIN_WIDTH]: {
+                width: `${spacings.TRIPLE}rem`,
+                height: `${spacings.TRIPLE}rem`,
+              },
+            },
+            time: {
+              marginLeft: `${spacings.FULL}rem`,
```

Comment:
Should this be `marginInlineStart` (or `marginInlineEnd`, can't remember!) to support RTL?

Reply:
<img width="517" alt="Screenshot 2025-07-08 at 13 18 40" src="https://github.com/user-attachments/assets/55ce09a5-53f4-4c60-b3dd-964145f131b5" />

seems to be ok